event/event.go

```diff
@@ -0,0 +1,50 @@
+package event
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"cloud.google.com/go/pubsub"
+	"github.com/bitrise-io/bitrise-step-analytics/models"
+)
+
+// Tracker ...
+type Tracker interface {
+	Send(analytics models.TrackEvent) error
+}
+
+type tracker struct {
+	topic   *pubsub.Topic
+	context *context.Context
+}
+
+func NewTracker(projectID string, topic string) Tracker {
+	ctx := context.Background()
+	client, err := pubsub.NewClient(ctx, projectID)
```

Comment:
this needs a json path exported with `GOOGLE_APPLICATION_CREDENTIALS` key to authenticate. We need to be careful with deployment

Reply:
I can do that potentially but we'll still need some tinkering around with heroku to have the json file there.
https://stackoverflow.com/questions/47446480/how-to-use-google-api-credentials-json-on-heroku